tests/e2e/scale/noobaa/test_scale_namespace_rpc.py

```diff
@@ -18,7 +18,7 @@ class TestScaleNamespace(E2ETest):
     Test creation of a namespace scale resource
     """
 
-    @skipif_ocs_version("<4.6")
+    @skipif_ocs_version("!=4.6")
```

Comment:
Why `!=4.6`? shouldn't this be `<4.6`?

Reply:
@keesturam rpc is supporting in 4.6+.  I just want to be aligned with manage team, since they are only run rpc on 4.6.

Reply:
don't we want to run this test in 4.7 or above?

Reply:
@keesturam AFAIK, this feature is not there in 4.6 i.e. in 4.6 we have this feature support using crd not with rpc.